src/app/auth.ts

```diff
@@ -1,21 +1,19 @@
-import SteamProvider, {PROVIDER_ID} from 'next-auth-steam'
+import SteamProvider, {STEAM_PROVIDER_ID} from 'next-auth-steam'
```

Comment:
This was deprecated, under the hood `PROVIDER_ID` is linked to `STEAM_PROVIDER_ID`